src/halmos/cheatcodes.py

```diff
@@ -289,12 +289,42 @@ def snapshot_state(
     return ByteVec(balance_hash + code_hash + storage_hash + path_hash)
 
 
-def create_calldata_contract(ex, arg, sevm, stack):
+def _get_contract_name(ex, arg) -> tuple[str | None, str | None]:
+    # TODO: support symbolic target using sevm.resolve_address_alias()
+    addr = con(
+        int_of(extract_bytes(arg, 4, 32), "symbolic address for SVM.createCalldata()"),
+        size_bits=160,
+    )
+    code = ex.code[addr]
```

Comment:
this probably fails with a KeyError if the address is not in code? consider something like

```suggestion
    code = ex.code.get(addr)
    if code is None:
        raise HalmosException(f"createCalldata: no contract at address {hexify(addr)})
```

Reply:
good catch!

Reply:
fixed: https://github.com/a16z/halmos/pull/517/commits/20b10edcff977be9e912947bca59f29e99a8282a